modules/phase_field/include/utils/EBSDAccessFunctors.h

```diff
@@ -21,8 +21,8 @@ class EBSDAccessFunctors
 public:
   /// Per element EBSD data point
   struct EBSDPointData {
-    Real phi1, phi, phi2, symmetry;
-    unsigned int grain, phase, op, global;
+    Real phi1, Phi, phi2;
```

Comment:
:barf: - This keeps getting uglier. Is there a reference you can put in here so that we can note why you are intentionally not following the code standards?